### ISSUE TITLE
The Envoy Part II

### ISSUE THREAD

**Issue by DaNarwhalChannel** [Wynncraft/Issues#8004]
# Delete this line if you've followed the README instructions and searched for a duplicate issue FIRST, and before posting any new issue.
I'm at the part where I come out of the cave. The tripwire triggers, and Maxie is meant to say something like "Psst! Over here!" But it doesn't even spawn him in. Help?

**Comment by KnixFice** [comment]
Im experiencing the same issue as well.

**Comment by LordHelmchen98** [comment]
Have you tried different servers? Or going back to Corkus City and then walking to the diner again?

**Comment by DaNarwhalChannel** [comment]
Yes, @LordHelmchen98 I have tried all of them, it still isn't spawning in.

**Comment by LordHelmchen98** [comment]
Are you still facing this?
